docs/userguide/campaigns.md

```diff
@@ -1,25 +1,156 @@
+from baybe.searchspace import SearchSpace
+
+cat_param = CategoricalParameter(name="cat_param", values=["a", "b", "c"])
+numdisc_param = NumericalDiscreteParameter(name="numdisc_param", values = tuple(range(1,11)))
+
+space = SearchSpace.from_product([cat_param, numdisc_param])
+```
+
+We refer to the [parameters](./parameters) resp. [search spaces](./searchspace) user
+guides for more details.
+
+For the optimization objective, we use a function taking values from 1 to 10 that we aim
+to maximize.
+
+```python
+from baybe.targets import NumericalTarget
+from baybe.objective import Objective
+
+target = NumericalTarget(name="quality", mode="MAX", bounds=(1,10))
+objective = Objective(mode="SINGLE", targets=[target])
+```
+
+Combining the search space and the objective now enables us to create a campaign.
+
+```python
+from baybe.campaign import Campaign
+
+campaign = Campaign(searchspace=space, objective=objective)
+```
+
+### Providing a different strategy
+
+By default, campaigns use the composite
+[`TwoPhaseStrategy`](baybe.strategies.composite.TwoPhaseStrategy).
+Using the optional `strategy` keyword, it is possible to provide a different
+[Strategy](baybe.strategies.base.Strategy).
+For more information on strategies in general, we refer to the corresponding
+[user guide](./strategy).
+
+### Creating a campaign via a configuration
+
+It is also possible to specify a `Campaign` via a configuration string and the function
+[`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+As fully specifying a configuration takes too much, we refer to the corresponding
+[example](./../../examples/Serialization/create_from_config).
+
+## Getting recommendations
+
+### Basics
+
+```{attention}
+Adding recommendations and measurements using the `recommend` and `add_measurements`
+functions is the safe only way to inform a `Campaign` object about new measurements.
+These functions update the necessary metadata that is crucial for the proper
+execution of a campaign. It is important to rely on these functions to maintain the
+integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+one argument, which is the `batch_quantity` keyword. This specifies the desired size of
+the batch of experiments to be conducted.
+
+```python
+rec = campaign.recommend(batch_quantity=3)
+```
+
+The `recommend` function returns a `DataFrame` with `batch_quantity` many rows, each 
+representing a set of parameters from the search space.
+
+### Influence of the batch size
+
+Regarding the choice of a suitable `batch_size`, it is important to understand the
+difference between performing multiple recommendations with batch size of 1 and a single
+recommendation with a larger batch size.
+* **Larger batch size**: When using a larger batch size, the recommended experiments are
```

Comment:
I like the overall text here, but 
i) Don't understand why you distinguish `smaller` and `larger`, the joint optimization kicks on from batch size > 1 so maybe a different classification here is better
ii) the mantra might not be true for special recommenders which do not do joint optimization (eg hybrid) or have special joint optimization like seq greedy